content/authorization.md

```diff
@@ -851,6 +851,25 @@ _NOTE_: The token response from a refresh will not contain a new
 refresh token.  The original refresh token from the initial
 grant response must be retained.
 
+#### Considerations for Handling 'online_access' ####
+
+When retrieving an access token utilizing using an
+online_access refresh, the most likely cause of failures
+is that the user's session has been terminated.  The
```

Comment:
Is there a specific error message that consumers would be able to look for to identify this is the case?

Reply:
All error codes follow the OAuth2 specifications, thus an `invalid_grant` is provided for both: https://datatracker.ietf.org/doc/html/rfc6749#section-5.2

Reply:
Read as a question, since I'm by no means the expert - would a reader of this end up understanding that info without us providing the details here?